tests/unit_tests/test_lobbyconnection.py

```diff
@@ -246,7 +246,7 @@ def test_command_admin_closeFA(mocker, lobbyconnection):
 
     protocol.send_message.assert_any_call({
         "command": "avatar",
-        "avatarlist": [{'url': 'http://content.faforever.com/faf/avatars/qai2.png', 'tooltip': 'QAI'}]
+        "avatarlist": [{'url': 'http://content.faforever.com/faf/avatars/qai2.png', 'tooltip': 'QAI'}, {'url': 'http://content.faforever.com/faf/avatars/UEF.png', 'tooltip': 'UEF'}]
```

Comment:
this has nothing to do with your PR or commit name, add one feature each time, is this causing a problem elsewhere?

Reply:
This is preventing a build entirely

Reply:
Then we should find out why that was missing and confirm that adding it is the way to go before just smuggeling into this commit.

Reply:
It is, it's because the test data contains 2 avatars and the server is waiting for only 1 to exist. As I've said on slack the best way to solve this is to compare the test result to what's inside the database (instead of this static string test), but for the moment I consider that to be enough. What broke this is that the test-data has been modified at one point which made this server impossible to build, something I've fixed a few days ago by editing the test data again. This test has simply become outdated.